extensions/amp-consent/0.1/cmps.js

```diff
@@ -76,3 +82,4 @@ CMP_CONFIG['Usercentrics'] = {
   'checkConsentHref': 'https://consents.usercentrics.eu/amp/checkConsent',
   'promptUISrc': 'https://amp.usercentrics.eu/amp.html',
 };
+
```

Comment:
Oops, looks like this our linter doesn't like this. Can you remove the new line?

Reply:
done: removed one new line